osu.Game/Screens/Play/HUD/DefaultRankDisplay.cs

```diff
@@ -45,6 +51,9 @@ private void load()
                     RelativeSizeAxes = Axes.Both
                 },
             };
+
+            if (skinEditor != null)
```

Comment:
I guess the alternative (which I would have chosen personally) is resolving `Player`. I'm probably fine with both, but did you consider this one?